src/components/SubscribeButton/NotificationModal.tsx

```diff
@@ -63,7 +64,7 @@ const CheckIcon = styled(Check)`
   bottom: 0;
   margin: auto;
 `
-const Input = styled.input<{ error: string }>`
+const Input = styled.input<{ borderColor: string }>`
```

Comment:
Using `$borderColor` is better, otherwise React will complain about invalid HTML prop borderColor 🤦

Reply:
noice, updated.